### PR TITLE
Prom exporter release

### LINKED ISSUE #263: prometheus-prefect-exporter helm chart
HI,

I'm using Prefect Platform everyday for our ML Engineers. First of all, thanks for spend time on improve Prefect and maintain helm charts.

In our case, we monitoring Prefect with Prometheus and visualizate metrics with Grafana, so we develop a simple exporter written in Python (currently migrate to Golang), but for now, works properly and provide us enough metrics to generate alerts if some flow-run failed, all deployments are paused, if work queues or work pools aren't available...

So, here are the repositories:

* prometheus-prefect-exporter: <https://github.com/devops-ia/prometheus-prefect-exporter> (written in Python) but I'm working to rewrite on Golang. Here's Docker releases: [docker releases](https://hub.docker.com/repository/docker/devopsiaci/prometheus-prefect-exporter/general)
* helm chart prometheus-prefect-exporter: <https://github.com/devops-ia/helm-charts/tree/main/charts/prometheus-prefect-exporter>
* dashboards: [prefect-platform-overview](https://grafana.com/grafana/dashboards/19806-prefect-platform-overview/) | [prefect-platform-flow-run-overview](https://grafana.com/grafana/dashboards/19809-prefect-flow-runs-overview/)

If you want, I can transferring a repository and open new PR on your new transfered repository. So, that's all... if you want speak or next-steps, I will be waiting. Thank you very much for everything.

### PR DESCRIPTION
**Pull request by jimid27**
- Resolves: https://github.com/PrefectHQ/prefect-helm/issues/263
- Releases Prometheus exporter when a specific tag is created prepended w/ `prefect-prometheus-exporter-`

### CODE REVIEW

File: .github/workflows/prometheus-exporter-release.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            --sign --key 'jamie@prefect.io' \
+            --keyring $SIGN_KEYRING \
+            --passphrase-file $SIGN_PASSPHRASE_FILE
+        env:
+          RELEASE_VERSION: ${{ steps.get_version.outputs.RELEASE_VERSION }}
+          PROMETHEUS_PREFECT_EXPORTER_VERSION: ${{ steps.get_version.outputs.PROMETHEUS_PREFECT_EXPORTER_VERSION }}
+          SIGN_KEYRING: ${{ env.SIGN_KEYRING }}
+          SIGN_PASSPHRASE_FILE: ${{ env.SIGN_PASSPHRASE_FILE }}
+
+      - name: Update chart index
```
[the remark this answers is not in the document]
Comment (jamiezieziula) [reply .github/workflows/prometheus-exporter-release.yaml:79]: i don't think so but idk!

File: .github/workflows/prometheus-exporter-release.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
+name: Release Prometheus Prefect Exporter Helm Chart
+
+"on":
+  push:
+    tags:
+      # prefect-prometheus-exporter-2023.9.1, but not prefect-prometheus-exporter-2023.09.01
+      # prefect-prometheus-exporter-2023.11.5, but not prefect-prometheus-exporter-2023.11.05
+      # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
+      - 'prefect-prometheus-exporter-2[0-9][2-9][3-9].1?[0-9].[1-3]?[0-9]'
```
Comment (jamiezieziula) [inline .github/workflows/prometheus-exporter-release.yaml:9]: this makes sense as the release name, is it also the tag name??

Also - should we convert to doing this for the other charts. i think we used to back in the day?

**Review by jamiezieziula — APPROVED** [review]
lgtm!

File: .github/workflows/prometheus-exporter-release.yaml (hunk 2 above)
Comment (jimid27) [reply .github/workflows/prometheus-exporter-release.yaml:9]: Yeah I'll create an issue to do the same w/ the other charts.  This is also the tag name, yes

**Pull request merged**